### ISSUE TITLE
Empty objects are being inserted into the messages array on the client side.

### ISSUE THREAD

**Issue by egropp** [microsoft/sample-app-aoai-chatGPT#357]
Empty objects are being inserted into the messages array on the client side.   This is causing lines like "role": message["role"]  in app.py to fail on followup chat questions.
![IssueWithEmptyObject](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/50458417/4f393563-6860-4dbd-a22f-8d216fd5594f)

**Comment by joonvena** [comment]
I did work around it by checking that message is not empty on line [https://github.com/microsoft/sample-app-aoai-chatGPT/blob/main/app.py#L402](402). 

```
...
for message in request_messages:
        if message:
            messages.append({
                "role": message["role"],
                "content": message["content"]
            })
...
```

**Comment by sarah-widder** [comment]
This was fixed in #372 , closing this issue for now.
